doc/examples/applications/plot_cornea_spot_inpainting.py

```diff
@@ -0,0 +1,98 @@
+"""
+============================================================
+Reconstruct dust-obstructed human cornea image with inpainting
```

Comment:
@lagru @ana42742 @decorouz I know we've already discussed the title at length, but I'm thinking:

* knowing that the tissue is *human* is interesting but I would prioritize *in-vivo* as a qualifying term;
* we can consider that we're *reconstructing* either the (entire) image (which is partly dust-obstructed) or the missing parts... Both may be accurate, depending on how we interpret 'reconstructing' / 'inpainting.'

What about:

* "Use inpainting to reconstruct dust-covered areas in in-vivo cornea image"
* "Inpaint dust-covered areas to reconstruct in-vivo cornea image"
* Replace "areas" with "regions" or "spots" in the above

?

Reply:
Your thoughts are valid. All the suggestions would make for a great title. The first suggestion have my vote.

> * "Use inpainting to reconstruct dust-covered areas in in-vivo cornea image"

I would replace "areas" with "regions".Since by definition, a region in an image is a group of connected pixels with similar properties. 
@ana42742 What do think?

Reply:
> by definition, a region in an image is a group of connected pixels with similar properties

I guess you're referring to the context where we measure the properties of given regions (cf. https://scikit-image.org/docs/dev/api/skimage.measure.html#skimage.measure.regionprops). It's definitely relevant to keep using terms in a similar way across the library (and, ideally, across the overall image processing community), so as to avoid confusions; this doesn't spare the need for a clear introduction at the beginning of each tutorial though.

I'm absolutely fine with the definition that a region of an image is a group of connected pixels, but the consideration about 'similar properties' only comes from applications (in practice, you're interested in a given region because it corresponds, for example, to those pixels where the intensity is highest).

Reply:
I was referring to a general definition of "regions" in the context of interpretation of an/any image. 

My comments is in agreement to your suggestion to replace "area" with "regions". I think "region" is a more appropriate term in the context of image applications.

Therefore:
Use inpainting to reconstruct dust_covered **areas** in in-vivo cornea image. 
Becomes; Use inpainting to reconstruct dust_covered **regions** in in-vivo cornea image.